src/components/VaultItemTiles.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import {
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  Card,
+  Box,
+} from "@mui/material";
+import DeleteTwoToneIcon from "@mui/icons-material/DeleteTwoTone";
+import NewItemForm from "./NewItemForm";
+import VaultItemPopOut from "./VaultItemPopOut";
+
+export default function VaultItemTiles({ items }) {
+  const itemsArray = Object.keys(items).map((item) => items[item].data);
+  const [open, openValue] = useState(false);
+  const [vaultItem, setValue] = useState(0);
+
+  const openPopup = (event, vaultItem) => {
+    setValue(vaultItem);
+    openValue(true);
+  };
+
+  function closePopup() {
```

Comment:
Any reason changing the function to this format compared to the openPopup? I would choose one way of defining a function to keep things consistent.